MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,567 @@
+imageEditor.ToolbarSettings.CropTypes = ImageCropType.Circle | ImageCropType.Square | ImageCropType.Free;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Customize default Effects option
+
+You can show the required effect types alone in the effects toolbar by using the `EffectTypes` property in the `ImageEditorToolbarSettings` class.
+
+{% tabs %}
+
+{% highlight XAML %}
+  
+<imageEditor:SfImageEditor Source="image.png">
+    <imageEditor:SfImageEditor.ToolbarSettings>
+        <imageEditor:ImageEditorToolbarSettings EffectTypes="Hue, Blur, Brightness" />
+    </imageEditor:SfImageEditor.ToolbarSettings>
+</imageEditor:SfImageEditor>
+      
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+imageEditor.ToolbarSettings.EffectTypes = ImageEffect.Hue | ImageEffect.Blur | ImageEffect.Brightness;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Customize default shape option
+
+You can show the required shape types alone in the shapes toolbar by using the `Shapes` property in the `ImageEditorToolbarSettings` class.
+
+{% tabs %}
+
+{% highlight XAML %}
+  
+<imageEditor:SfImageEditor Source="image.png">
+    <imageEditor:SfImageEditor.ToolbarSettings>
+        <imageEditor:ImageEditorToolbarSettings Shapes="Circle, Line, DottedArrow, DoubleArrow" />
+    </imageEditor:SfImageEditor.ToolbarSettings>
+</imageEditor:SfImageEditor>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+imageEditor.ToolbarSettings.Shapes = AnnotationShape.Circle | AnnotationShape.Line | AnnotationShape.DottedArrow | AnnotationShape.DoubleArrow;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## To customize the ColorPalette
+
+You can change the default colors of the `ColorPalette` in toolbar.
+
+{% tabs %}
+
+{% highlight XAML %}
+  
+<imageEditor:SfImageEditor Source="image.png">
+    <imageEditor:SfImageEditor.ToolbarSettings>
+        <imageEditor:ImageEditorToolbarSettings>
+            <imageEditor:ImageEditorToolbarSettings.ColorPalette>
+                <Color>Yellow</Color>
+                <Color>Pink</Color>
+                <Color>Violet</Color>
+            </imageEditor:ImageEditorToolbarSettings.ColorPalette>
+        </imageEditor:ImageEditorToolbarSettings>
+    </imageEditor:SfImageEditor.ToolbarSettings>
+</imageEditor:SfImageEditor>
+      
+{% endhighlight %}
+
+{% highlight C# %}
+
+    SfImageEditor imageEditor = new SfImageEditor();
+    imageEditor.Source = ImageSource.FromFile("image.png");
+    imageEditor.ToolbarSettings.ColorPalette.Add(Colors.Yellow);
+    imageEditor.ToolbarSettings.ColorPalette.Add(Colors.Pink);
+    imageEditor.ToolbarSettings.ColorPalette.Add(Colors.Violet);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+N> Color palette colors are common for Text, Shape and Pen.
+
+## Default Toolbar items
+
+### Toolbar item
```

Comment:
### Toolbar item ,  ### Name remove just explain

Reply:
Removed and just explain a topic